### PR TITLE
Add to footer Discord Icon

### PR DESCRIPTION
**Pull request by devbold**
### Add to footer Discord Icon
-Add to footer Discord Icon

rcn-loans--requested-00

**Branch:** footer-addDiscord
**Refe:** 
https://discordapp.com/

### CODE REVIEW

File: src/app/core/social-container/social-container.component.html [hunk 1]
```diff
@@ -7,7 +7,8 @@
       <li><a href="https://www.linkedin.com/company/ripio-credit-network/" matTooltip="Linkedin" matTooltipPosition="above" target="_blank"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
       <li><a href="https://www.reddit.com/r/rcn_token/" matTooltip="Reddit" matTooltipPosition="above" target="_blank"><i class="fab fa-reddit fa-lg"></i></a></li>
       <li><a href="https://github.com/ripio/rcn.loans" matTooltip="Github" matTooltipPosition="above" target="_blank"><i class="fab fa-github fa-lg"></i></a></li>
-      <li><a href="{{ linkContract }}" matTooltip="Github" matTooltipPosition="above"><i class="fas fa-link fa-lg"></i></a></li>
+      <li><a href="https://discordapp.com/channels/524313273512755211/524313273512755217" matTooltip="Discord" matTooltipPosition="above" target="_blank"><i class="fab fa-discord fa-lg"></i></a></li>
+      <!-- <li><a href="{{ linkContract }}" matTooltip="Github" matTooltipPosition="above"><i class="fas fa-link fa-lg"></i></a></li> -->
```
Comment (Agusx1211) [inline src/app/core/social-container/social-container.component.html]: Why is the Github link commented out?

File: src/app/core/social-container/social-container.component.html [hunk 2]
```diff
@@ -7,7 +7,8 @@
       <li><a href="https://www.linkedin.com/company/ripio-credit-network/" matTooltip="Linkedin" matTooltipPosition="above" target="_blank"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
       <li><a href="https://www.reddit.com/r/rcn_token/" matTooltip="Reddit" matTooltipPosition="above" target="_blank"><i class="fab fa-reddit fa-lg"></i></a></li>
       <li><a href="https://github.com/ripio/rcn.loans" matTooltip="Github" matTooltipPosition="above" target="_blank"><i class="fab fa-github fa-lg"></i></a></li>
-      <li><a href="{{ linkContract }}" matTooltip="Github" matTooltipPosition="above"><i class="fas fa-link fa-lg"></i></a></li>
+      <li><a href="https://discordapp.com/channels/524313273512755211/524313273512755217" matTooltip="Discord" matTooltipPosition="above" target="_blank"><i class="fab fa-discord fa-lg"></i></a></li>
```
Comment (Agusx1211) [inline src/app/core/social-container/social-container.component.html]: I am not sure if `https://discordapp.com/channels/524313273512755211/524313273512755217` is a valid invite link, maybe we can use `https://discord.gg/62UCxbS`

File: src/app/core/social-container/social-container.component.html (hunk 1 above)
Comment (devbold) [reply src/app/core/social-container/social-container.component.html]: There a mistake on the tooltip message

**Comment by Agusx1211** [comment]
Closes #182

**Pull request merged**